<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscoveryMainInterface</class>
 <widget class="QWidget" name="DiscoveryMainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 219, 97);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 219, 97);
color: rgb(0, 0, 0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="2,8">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout" rowstretch="3,6,6,1">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8,2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ButtonAdd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
font: 22pt &quot;微软雅黑&quot;;
background-color: rgb(37, 37, 37);
color: rgb(249, 219, 97);
border-radius:15px;</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineEdit_Serch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:20px;
border-color: rgb(145, 145, 145);
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 204);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonQuery">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
background-color: rgb(37, 37, 37);
color: rgb(249, 219, 97);
border-radius:15px;</string>
            </property>
            <property name="text">
             <string>查找</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="MenuButtonone">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
background-color: rgb(37, 37, 37);
color: rgb(249, 219, 97);
border-radius:15px;</string>
            </property>
            <property name="text">
             <string>关于</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MenuButtonTwo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
background-color: rgb(37, 37, 37);
color: rgb(249, 219, 97);
border-radius:15px;</string>
            </property>
            <property name="text">
             <string>菜单二</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MenuButtonThree">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
background-color: rgb(37, 37, 37);
color: rgb(249, 219, 97);
border-radius:15px;</string>
            </property>
            <property name="text">
             <string>菜单三</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="MenuButtonFour">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
font: 18pt &quot;微软雅黑&quot;;
background-color: rgb(37, 37, 37);
color: rgb(249, 219, 97);
border-radius:15px;</string>
            </property>
            <property name="text">
             <string>菜单四</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="DetialTableview" name="tableView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DetialTableview</class>
   <extends>QTableView</extends>
   <header location="global">BaseClass/detialtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
